rancher/installing-rancher/installing-server/multi-nodes/index.md

```diff
@@ -6,109 +6,55 @@ layout: rancher-default
-
-    ```sql
-    CREATE DATABASE IF NOT EXISTS cattle COLLATE = 'utf8_general_ci' CHARACTER SET = 'utf8';
-    GRANT ALL ON cattle.* TO 'cattle'@'%' IDENTIFIED BY 'cattle';
-    GRANT ALL ON cattle.* TO 'cattle'@'localhost' IDENTIFIED BY 'cattle'; 
-    ```
-
-5. Launch your rancher server instances with this command:
-      
-      ```bash
-      sudo docker run -d --restart=always -p 8080:8080 \
-        -e CATTLE_DB_CATTLE_MYSQL_HOST=<hostname or IP of MySQL instance> \
-        -e CATTLE_DB_CATTLE_MYSQL_PORT=<port> \
-        -e CATTLE_DB_CATTLE_MYSQL_NAME=<Name of Database> \
-        -e CATTLE_DB_CATTLE_USERNAME=<Username> \
-        -e CATTLE_DB_CATTLE_PASSWORD=<Password> \
-        -e DEFAULT_CATTLE_MACHINE_EXECUTE=false \
-        -e CATTLE_HOST_API_PROXY_MODE=ha \
-        -e CATTLE_HOST_API_PROXY_HOST=<host_ip:port that is accessible from your browser and to all compute nodes> \
-        -e CATTLE_ZOOKEEPER_CONNECTION_STRING=<comma separated list of zookeeper IPs ie. 10.0.1.2,10.0.1.3> \
-        -e CATTLE_REDIS_HOSTS=<comma separated list of host:port server ips. ie 10.0.1.3:6379,10.0.1.4:6379> \
-        -e CATTLE_REDIS_PASSWORD=<optional Redis password> \
-        rancher/server
-      ```  
-       > **Note:** In order to start websocket-proxy, Rancher server needs to be up and running. Therefore, Rancher server needs to be started with the `CATTLE_HOST_API_PROXY_MODE` and `CATTLE_HOST_API_PROXY_HOST` variables to be able to use websocket-proxy.
-      
-6. Point your load balancer at the server targets
-7. Go to new installation ip: `http://<LB ip>:<port>`
-8. Bring up the websocket-proxy:
-   * download the Public Key to verify host tokens:
-     * `curl -X GET -O http(s)://<rancher>/v1/scripts/api.crt`
-   * Bring up websocket-proxy running on the host. (Do not use localhost)
-     * `docker run -d -p <port>:<port> -v $(pwd)/api.crt:/api.crt rancher/server websocket-proxy -jwt-public-key-file=/api.crt -listen-address=0.0.0.0:<port>` 
-9. Bring up `go-machine-service`
-   * Create a service account and API keys:
-      * Visit `http(s)://<rancher_server/v1/accounts` in a web browser.
-      * Click `+Create` button.
-          - kind: service
-          - name: RemoteMachineService
-          - uuid: RemoteMachineService
-      * Click `Show Request` then `Send Request`
-      * Click `Follow Self Link`
-      * In the body there is a links section with a credentials link, click it.
-      * Click `+Create`
-      * Click `Show Request`
-      * Click `Follow Self Link`
-      * Make note of publicValue and secretValue.
-   * Launch `go-machine-service` with this command:
-      
-      ```bash
-      sudo docker run -d --restart=always \
-        -e CATTLE_URL=http(s)://<Rancher URL>/v1 \
-        -e CATTLE_ACCESS_KEY=<Service accounts publicValue> \
-        -e CATTLE_SECRET_KEY=<Service accounts secretValue> \
-        rancher/server go-machine-service
-	  ```	
+* MySQL Database with at least 1 GB RAM
```

Comment:
Plan for 50 Connections per Rancher Management node. (So 3 nodes needs to support at least 150 connections) This is helpful when selecting RDS or other cloud instances.